client/src/components/AddProjectForm/index.js

```diff
@@ -71,7 +77,7 @@ export default class AddProjectForm extends Component {
 
     return (
       <div className={styles.container}>
-        {showForm && this.renderForm() }
+        {showForm && this.renderForm(intl) }
```

Comment:
Let's get rid of renderForm and just put { showForm && <div className={styles.form}>... here.